app/src/main/java/com/f0x1d/logfox/ui/activity/CrashDetailsActivity.kt

```diff
@@ -35,18 +38,22 @@ class CrashDetailsActivity: BaseViewModelActivity<CrashDetailsViewModel, Activit
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-
+        setTitle(R.string.crash_details)
```

Comment:
But no `setTitle` in `MainActivity`? It is not needed there?

Reply:
The talkback speaks the activity title when entering a new activity, so the title needs to be set here, otherwise the `label` in `AndroidManifest.xml` will be used I now think I need to move it within the xml rather than within the code. For the `fragment ` in `MainActivity` I don't know how to mess with it for now.